### PR TITLE
Lazy load on objects

### PR DESCRIPTION
**Pull request by maxcapodi78**

### CODE REVIEW

**Review by Samuelopez-ansys — DISMISSED** [review]
LGTM

**Review by Samuelopez-ansys — APPROVED** [review]

File: pyaedt/application/Design.py [hunk 1]
```diff
@@ -262,8 +263,11 @@ def load_aedt_thread(path):
         self._logger.odesign = self.odesign
         AedtObjects.__init__(self, is_inherithed=True)
         self.logger.info("Aedt Objects correctly read")
-        if t:
-            t.join()
+        # if t:
+        #     t.join()
```
Comment (Alberto-DM) [inline pyaedt/application/Design.py:267]:
```suggestion

```

File: pyaedt/modeler/cad/Primitives.py [hunk 2]
```diff
@@ -6596,9 +6784,9 @@ def get_object_from_name(self, objname):
             3D object returned.
 
         """
-        if objname in self._object_names_to_ids:
-            object_id = self.get_obj_id(objname)
-            return self.objects[object_id]
+        if objname in self.object_names:
+            # object_id = self.get_obj_id(objname)
```
Comment (Alberto-DM) [inline pyaedt/modeler/cad/Primitives.py:6788]:
```suggestion

```

File: _unittest/conftest.py [hunk 3]
```diff
@@ -94,7 +94,7 @@
 desktop_version = config["desktopVersion"]
 new_thread = config["NewThread"]
 settings.use_grpc_api = config["use_grpc"]
-
+settings.objects_lazy_load = False
```
Comment (Alberto-DM) [inline _unittest/conftest.py:97]: Is it not tested? I think we should run at least Primitives test with and without lazy load

**Review by Alberto-DM — COMMENTED** [review]
LGTM, just a couple of comments

**Pull request merged**